msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1524,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+## Receive meeting participant events
+
+Your bot can receive real-time meeting details such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events. For example, if you want your bot app to notify or return the time stamp when an attendee joined or left the meeting, subscribe to these participant events in Developer Portal.
```

Comment:
This section should be simplified and be clear. Maybe use the above paragraph instead?

A bot can receive real-time notifications such as participant join and leave events only if it is subscribed to these events. To subscribe, the bot owner needs to go to the Developer Portal.

Reply:
Updated